server/handlers/handlers.go

```diff
@@ -17,4 +17,8 @@ func NewHandlers(gr interfaces.Router, db interfaces.MongoInterface) {
 	gr.AddRoute("/api/example_json_response", http.MethodGet, handlers.ExampleJsonReponse)
 	gr.AddRoute("/api/save_new_user", http.MethodPost, handlers.SaveNewUser)
 	gr.AddRoute("/api/login_request", http.MethodPost, handlers.LoginRequest)
+	gr.AddRoute("/api/google", http.MethodGet, handlers.Google)
```

Comment:
For now we can use this page to log in by google, maybe in the future we can have an option for log in through Google in the index page